ios/FluentUI/Date Time Pickers/Date Picker/DatePickerController.swift

```diff
@@ -195,6 +193,14 @@ class DatePickerController: UIViewController, GenericDateTimePicker {
         }
     }
 
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if let segmentedControl = segmentedControl, previousTraitCollection?.userInterfaceStyle != traitCollection.userInterfaceStyle {
+            segmentedControl.style = (traitCollection.userInterfaceStyle == .dark) ? .onBrandPill : .primaryPill
```

Comment:
This is interesting. Shouldn't there be a new style to cover these variations of colors between dark/light modes?
(since there's a valid requirement for these different styles)

Reply:
The segmented control supports custom colors, so we probably don't need a third style that's onBrand light and primary dark. The onBrand style would still have a colorful background in dark mode.

Reply:
@huwilkes I understand that it's technically possible either through custom colors or the way it's currently implemented here (setting the style based on the trait collection).
My question is more about the meaning of this variation of "primary for light and onBrand for dark".
If it's a scenario that's part of the design specs (toolkit), there should probably be a standardized way of using it instead of just setting custom colors.

Reply:
yeah, I went back and forth about actually exposing the style property of the segment control to public. it seemed bit overkill to have another style which is practically the same color as existing style. I think because segment control is mostly focused on large title navigation bar but not really for this "edge" case of date picker which the segment control shouldn't take too much attention compared to other ui.